src/main/java/nextstep/courses/domain/base/BaseInfo.java

```diff
@@ -0,0 +1,30 @@
+package nextstep.courses.domain.base;
+
+import java.time.LocalDateTime;
+
+public class BaseInfo {
+
+  private Long creatorId;
```

Comment:
```suggestion
  private final Long creatorId;
```

인스턴스 변수는 기본  final로 구현하는 습관을 들이는 것은 어떨까?